docs/setup.md

```diff
@@ -1,10 +1,52 @@
 # Local Development Environment
 
-## Requirements
+__NOTE: Documentation assumes you are using MacOS with at least 8GB of memory on an Intel, M1, or M2 processor__
+
+:clap: Thank you for considering to contribute to the YaleSites Platform!
+
+## Requirements & Setup
+
+1. [Contact YaleSites](mailto:yalesites@yale.edu?subject=Developer%20Access%20Request) for access to resources such as Pantheon, Figma designs, etc.
+2. [GitHub](#github)
+   1. SSH authorization key
+   2. [Github Package Personal Access Token](https://github.com/yalesites-org/component-library-twig) in environment variable `YALESITES_BUILD_TOKEN`
```

Comment:
Why does this one link to the Component Library? Is that what requires the build token? I thought that the token was required for the main repo too? (I forget!)

Reply:
Yes, that is where the current documentation was to generate the token.  I had thought about duplicating it, but instead linked to it so we wouldn't have to keep it in sync.  I also thought that I should update this to add (see prerequisites) if I continue to link to component library.

Yes it is needed here as well as it uses the component library in the setup.  Do you think it'd be bad to migrate that over to this document and then reference it from the component library README?  That then make this more of the source of truth in setup.

Reply:
Thinking about it more, I think we may attempt to duplicate how to do this here as it is pretty important to getting up and running, and I know I'd hate being redirected there and then have to realize prerequisites are collapsed.  Thanks for bringing that up and speaking about it.

Reply:
Added, and I feel better when clicking that link now.